src/tactic/monotonicity/basic.lean

```diff
@@ -96,6 +96,7 @@ meta def mono_head_candidates : ℕ → list expr → expr → tactic mono_key
 meta def monotoncity.check (lm_n : name) (prio : ℕ) (persistent : bool) : tactic mono_key :=
```

Comment:
```suggestion
meta def monotonicity.check (lm_n : name) (prio : ℕ) (persistent : bool) : tactic mono_key :=
```

Reply:
Thanks Scott, I haven't use the suggestion since uses of this definition also needed to be changed, but the intent is the same.